static_content/index.html

```diff
@@ -13,19 +13,19 @@
     do stuff
   </button>
 
-  <button onclick="doAPICall('http://127.0.0.1:5000/introduction/')">
+  <button onclick="doAPICall('http://localhost:5000/introduction/')">
```

Comment:
_After_  changing all the other stuff, I noticed that by swapping `127.0.0.1` for the equivalent `localhost`, I removed the last place where a CORS issue existed. Cool!